portfolio/src/main/java/com/google/sps/servlets/buildtable.java

```diff
@@ -0,0 +1,54 @@
+package com.google.sps.servlets;
+
+import com.google.cloud.datastore.*;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery.OrderBy;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for listing tasks. */
+@WebServlet("/buildtable")
+public class buildtable extends HttpServlet {
+    private String convertToJsonUsingGson(ArrayList<String> x) {
+        Gson gson = new Gson();
+        String newt = gson.toJson(x);
+        return newt;
+    }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    Query<Entity> query =  Query.newEntityQueryBuilder().setKind("Bask")
+            .setOrderBy(OrderBy.desc("TimestampForMusic"))
+            .build();
+            //Query<Entity> query = new Query<Entity>("Music Selection");
```

Comment:
remove dead code

Reply:
Okay, will do. And i apologize about the size of the pull request - back in week 2 when I made this, i didn't know that pull requests were supposed to be small, lol